lib/membrane_wav/parser.ex

```diff
@@ -1,49 +1,60 @@
 defmodule Membrane.WAV.Parser do
   @moduledoc """
-    Element responsible for parsing WAV files.
+  Element responsible for parsing WAV files.
 
-    It parses only PCM / uncompressed format. In case of different format, error is raised.
-    ```
-                                        WAV Header
+  It requires WAV file in uncompressed, PCM format on the input (otherwise error is raised) and
+  provides raw audio on the output. WAV header is parsed to extract metadata for creating Caps.
```

Comment:
I guess capitalization on caps is not needed here as in this context it's not a module name
```suggestion
  provides raw audio on the output. WAV header is parsed to extract metadata for creating caps.
```